### DISCUSSION TITLE
add settings menu

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#220] [Feature Requests]
a settings menu would be nice, It would have options such as to reduce animations, change the taskbar size, color, start icon, etc. It would also be nice if it was based on windows 10 because it seems that the desktop OS is based on windows 10.

**Comment by DustinBrett** [comment]
Thanks for the suggestion. I for sure would like to have more customization, and long term that is the plan. But as it is now, there is a lot that is hard coded in a way where doing something like changing the size of the taskbar could make apps act funny that expect it to be a certain height. I'd like to eventually have a full on "Control Panel".
